Real-Dev-Squad/website-dashboard
# Feat/audit log

### Developer Name: Joy Gupta

### PR Number(s): 573

### Issue Ticket Number:
- https://github.com/Real-Dev-Squad/website-dashboard/issues/494

### Backend Changes

### Frontend Changes

### Is Under Feature Flag

### Database changes: 

### Breaking changes (If your feature is breaking/missing something please mention pending tickets): 

### Deployment notes:
NA

### Tested in staging

### List of PRs going in this Sync
- https://github.com/Real-Dev-Squad/website-my/pull/503
- https://github.com/Real-Dev-Squad/website-backend/pull/1592

### Proposal Doc
- https://docs.google.com/document/d/1sf-BAs7C6wmfjRVS-82X492hcb_7jAfIM-7pWtizuXE/edit

### QA Instructions, Screenshots, Recordings

Run `yarn test` and there you can see test case running.
![image](https://github.com/Real-Dev-Squad/website-dashboard/assets/56365512/b6896e06-813b-4d7c-ab90-c5aeba92e028)


https://github.com/Real-Dev-Squad/website-dashboard/assets/56365512/12b34e9b-e1ea-425a-8bd0-a56b22b867dd
